src/validation_workflow/docker/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+---
+version: '3'
```

Comment:
No need to add another docker-compose file as in `docker/release` we already have compose file for different versions. 

The current one you include wont work on 1.x anyway.

Reply:
Thanks for notifying that, @peterzhuamazon, I took the docker-compose.yml from https://opensearch.org/samples/docker-compose.yml and put some placeholders in the file to be replaced with the arguments where taken from the user input. Also, placeholder for the container_name which avoid the possible name conflict if the user also have the same container name existing at the time the script is running; hence, I wanted to have this edited docker-compose.yml to be used for validation specifically. 

Good point you mentioned that 'won't work on 1.x anyway'.. I wasn't aware of it, does that mean the docker-compose.xml provided at opensearch.org actually does not work on 1.x either? should we notify the documentation team about that as well?

Reply:
Hi, @peterzhuamazon , I added another line to replace 'initial_cluster_manager_nodes' with 'initial_master_nodes' in docker-compose.xml to get it work on ver1.x, Thanks again for noticing me that the template earlier did not work on 1.x. I did not find it mentioned on https://opensearch.org/ tho ... maybe the documentation team should note that there just to be clear to the users.

Reply:
Is it possible to just retrieve the docker-compose file on the go? The reason being we won't know what is shipped to customer and we would need to maintain docker-compose at multiple places. Dont worry about container name, let if fail if one already exists. You can grab the latest docker-compose yml from website and just replace the version.

Reply:
Thanks for the idea, @gaiksaya .. I think so, I can just fetch the file from https://opensearch.org/samples/docker-compose.yml at everytime script run and make some necessary replacement. Do you think we can do that as an enhancement later on?

Reply:
Sure! Make sure you create an issue and have a fast follow up later.
Thanks!

Reply:
Yes, so far, I collected 3 items to enhance at https://github.com/opensearch-project/opensearch-build/issues/3205  :) thanks.

Reply:
Hi. @gaiksaya , I got the script updated to fetch the docker-compose.xml on the go, and also removed the local file. thanks! https://github.com/opensearch-project/opensearch-build/pull/3038/commits/cfe77608ed8fc09022387d6ee8b5f87f754172e4

Reply:
Like I mentioned in the comment why retrieve the file from website when our repo in `docker/release` already have the necessary files?

Reply:
Because we want to validate what customer uses not what we have in the repo.

Reply:
I went to check if any existing code that is using the /docker/release/dockercomposefiles/xx but did not find any; hence, I thought it would be better Not to link the docker-validation to it either as well. ( keeping no dependancy with those twp yml files as current )
By fetching the docker-compose.yml from the example file provided by https://opensearch.org/ looks a good suggestion to me because it is the file brocasted to all in official, and we don't need to maintain it either; hence, I switch to use it. Hope that make sense... thanks!